scipy/signal/_peak_finding.py

```diff
@@ -293,13 +308,13 @@ def peak_prominences(vector, peaks, wlen=None):
     Find all peaks and calculate prominences
 
     >>> from scipy import signal
-    >>> peaks = signal.argrelmax(vector)[0]
+    >>> peaks = signal.find_peaks(vector)
```

Comment:
what is `vector`?

Reply:
The array to be searched for peaks. Defined in line 306 (second row) in the folded section above.